kolibri/auth/api.py

```diff
@@ -113,7 +113,7 @@ def list(self, request):
         elif type(logged_in_user) is AnonymousUser:
             return Response(Facility.objects.all().values_list('id', flat=True))
         else:
-            return Response(logged_in_user.facility_id)
+            return Response([logged_in_user.facility_id])
```

Comment:
can logged_in_user. facility_id includes multiple ids? On the auth.models level, a FacilityUser can associated with multiple facilities.